### DISCUSSION TITLE
12 hours time format

### DISCUSSION THREAD

**Discussion by MarkTarol** [mathoudebine/turing-smart-screen-python#85] [Q&A] [answered]
is there a way to change the time format to 12 hours?

**Comment by hunterm13** [comment]
I'm not sure if you still need the answer but the way I did it was going into the simple-program.py and adding .strftime() to the end of datetime.now() inside of DisplayText()

**Comment by mathoudebine** [comment, marked as answer]
For the system-monitor (main.py) the date & time should now be displayed based on your computer locale
